src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2/IntegrationRuntimes/RemoveAzureDataFactoryIntegrationRuntimeCommand.cs

```diff
@@ -26,6 +27,14 @@ namespace Microsoft.Azure.Commands.DataFactoryV2
         SupportsShouldProcess = true)]
     public class RemoveAzureDataFactoryIntegrationRuntimeCommand : IntegrationRuntimeCmdlet
     {
+        [Parameter(Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeLinks)]
+        public SwitchParameter Link { get; set; }
+
+        [Parameter(Mandatory = false,
+            HelpMessage = Constants.HelpLinkedFactoryName)]
+        public string LinkedDataFactoryName { get; set; }
```

Comment:
Should this accept an array if you want to delete multiple linked data factories?

Reply:
No, this feature only accepts single factory. It is the same on the backend too.

Reply:
@zhangyd2015

Reply:
No, by design only one single linked data factory would be deleted.